### ISSUE TITLE
Follow the guide but fail to run example test_online_beamforming.py

### ISSUE THREAD

**Issue by leo0730mao** [kkumatani/distant_speech_recognition#11]
I follow the document to install btk20 and everything goes well, but failed to run the example:

python2 test_online_beamforming.py \                                                                      maoyue@maoyuedeMBP
      -c confs/gscrls.json \
      -i data/CMU/R1/M1005/KINECT/RAW/segmented/U1001_1M_16k_b16_c1.wav \
         data/CMU/R1/M1005/KINECT/RAW/segmented/U1001_1M_16k_b16_c2.wav \
         data/CMU/R1/M1005/KINECT/RAW/segmented/U1001_1M_16k_b16_c3.wav \
         data/CMU/R1/M1005/KINECT/RAW/segmented/U1001_1M_16k_b16_c4.wav \
      -o out/U1001_1M_gscrls.wav
Fatal Python error: PyThreadState_Get: no current thread
[1]    92974 abort      python2 test_online_beamforming.py -c confs/gscrls.json -i     -o

I think a possible cause is multi-version python existing, but I don't know how to fix this.

**Comment by kkumatani** [comment]
Yes, this seems to happen if your python environment has multiple versions as reported in https://github.com/kkumatani/distant_speech_recognition/issues/4
The quick solution is make sure your environment does not mix python 2 & 3. Otherwise, you have to wait for btk to support python 3.
